projects/batfish/src/org/batfish/grammar/cisco/CiscoParser.g4

```diff
@@ -2701,14 +2705,21 @@ t_server
 
 t_address
 :
+   ADDRESS
    (
-      ADDRESS ((IPV4 IP_ADDRESS) | (IPV6 IPV6_ADDRESS)) NEWLINE
-   )
+      (
+         IPV4 IP_ADDRESS
+      )
+      |
+      (
+         IPV6 IPV6_ADDRESS
+      )
+   ) NEWLINE
 ;
 
 t_key
 :
-    KEY DIGIT HEX_DIGIT+
+   KEY DEC VARIABLE NEWLINE
```

Comment:
I don't believe this change will work if the key happens to only include digits from 0-9 (eschewing A-F). Please add a unit test with such key. If it breaks, you will likely have to use lexer modes to solve this.